data_steward/cdr_cleaner/cleaning_rules/ehr_submission_data_cutoff.py

```diff
@@ -74,28 +96,8 @@ def __init__(self,
                          affected_datasets=[cdr_consts.UNIONED],
                          project_id=project_id,
                          dataset_id=dataset_id,
-                         sandbox_dataset_id=sandbox_dataset_id)
-
-    def get_affected_tables(self):
-        """
-        This method gets all the tables that are affected by this cleaning rule which are all the CDM tables
-            except for the person table. The birth date field in the person table will be cleaned in another
-            cleaning rule where all participants under the age of 18 will be dropped. Ignoring this table will
-            optimize this cleaning rule's runtime.
-
-        :return: list of affected tables
-        """
-        tables = []
-        for table in AOU_REQUIRED:
-
-            # skips the person table
-            if table == 'person':
-                continue
-
-            # appends all CDM tables except for the person table
-            else:
-                tables.append(table)
-        return tables
+                         sandbox_dataset_id=sandbox_dataset_id,
+                         affected_tables=get_affected_tables())
```

Comment:
This works. We can also change this line to 
`affected_tables=[table for table in AOU_REQUIRED if table != 'person']`
and remove `get_affected_tables()` altogether.

If we do that, patching for the test will be similar to this one: https://github.com/all-of-us/curation/blob/develop/tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/remove_records_with_wrong_date_test.py#L56